integrations/integration-vip-config.php

```diff
@@ -61,7 +61,11 @@ public function __construct( string $slug ) {
 	 * @param string $slug A unique identifier for the integration.
 	 */
 	private function set_config( string $slug ): void {
-		$config = $this->get_vip_config_from_file( $slug );
+		if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) && 'local' === constant( 'VIP_GO_APP_ENVIRONMENT' ) ) {
+			$config = $this->get_local_config_from_file( $slug );
+		} else {
```

Comment:
Do we need this? We can define the `WPVIP_INTEGRATIONS_CONFIG_DIR` in Dev Environments as well. This will make it possible to reuse this code in Codespaces.

Reply:
Or maybe something like this (WordPress way):

```php
$config = apply_filters( 'integrartions_load_config', null, $slug );
if ( is_null ( $config ) ) {
    $config = $this->get_vip_config_from_file( $slug );
}

$config = apply_filters( 'integrations_post_process_config', $config );
```

Reply:
Filter names are, of course, to be discussed :-)

If we use this approach, we can delegate the loading logic to the `dev-env-plugin.php`: this will work both in dev environments and codespaces (the current `get_local_config_from_file()` implementation will not work with codespaces);

The `post_process` filter may be used to filter out some data (I am not sure this is necessary though but I like to have such a possibility).

Reply:
Well, each config is split per file and is a PHP snippet, I don't think we want that in local env?